x/gamm/keeper/pool.go

```diff
@@ -50,7 +52,8 @@ func (k Keeper) NewPool(ctx sdk.Context, poolId uint64, poolParams types.PoolPar
 	return poolAcc, nil
 }
 
-func (k Keeper) GetNextPoolNumber(ctx sdk.Context) uint64 {
+// getNextPoolNumber returns the next pool number
+func (k Keeper) getNextPoolNumber(ctx sdk.Context) uint64 {
```

Comment:
Why do you want this to be private?

Reply:
Wanted to reduce the API surface of the keeper

Reply:
Oh I didn't realize it also incremented the value, I thought it just peeked.  Yeah, then should make it private.